lib/appsignal/config.rb

```diff
@@ -147,6 +151,10 @@ def config_file
         root_path.nil? ? nil : File.join(root_path, 'config', 'appsignal.yml')
     end
 
+    def detect_from_system
+      self[:running_in_container] = true if Appsignal::System.container?
```

Comment:
Right now it returns a "truthy" value. false/nil or true/string. It can return an id from `Appsignal::System::Container.id` That's why I wrapped it in an if.

I suggest we leave it like this right now, and when I get back to it, try to have this method return boolean-only values.

Right now I fetch the container_id since it's useful information to add to the transaction metadata. It's not added to the metadata yet.